tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -118,7 +114,6 @@ public override void Dispose()
             }
 
             _httpClient?.Dispose();
```

Comment:
in this similar [PR ](https://github.com/DataDog/dd-trace-dotnet/pull/3340)I started before vacations ,  I identified the httpClient.Dispose() as different from the` ClrProfilerIntegrations.Integrationtests` and was wondering if it's not that too, as httpClient is a bit peculiar under the scenes regarding disposing 🤔 
would need to run the pipeline several times on these branches to see..

Reply:
But the client is to send requests to the sample. The socket exception occurs when the sample communicates with the agent.

Reply:
true it's probably nothing to do with it then